packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -33,6 +35,21 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
   name: 'JiraServerIntegration',
   description: 'Jira Server integration data for a given team member',
   fields: () => ({
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'Composite key in jiraServer:providerId format',
+      resolve: async ({teamId, userId}: {teamId: string; userId: string}, _args, {dataLoader}) => {
+        const auth = await dataLoader
+          .get('teamMemberIntegrationAuths')
+          .load({service: 'jiraServer', teamId, userId})
+
+        if (!auth) {
```

Comment:
+1 I wonder whether the `id` type should be nullable and we return null if there's no `auth`? Otherwise, an empty string might be difficult to debug